oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -288,7 +288,24 @@ This section summarizes the requirements for long-term developer documentation.
 Format
 ~~~~~~
 
-We will use `Restructured Text (rST)`_ for all developer documentation.
+We will use `reStructured Text (rST)`_ for all developer documentation.
```

Comment:
Since the rationale was added in this section, can you add a back reference from the `Markdown Format` section to this one?   We can then remove the more shallow statement of `Here is an interesting blog post on..` in that section.